pandas/tests/arrays/masked/test_arrow_compat.py

```diff
@@ -36,9 +40,7 @@ def test_arrow_roundtrip(data):
     table = pa.table(df)
     assert table.field("a").type == str(data.dtype.numpy_dtype)
 
-    msg = "Passing a BlockManager to DataFrame is deprecated"
-    with tm.assert_produces_warning(DeprecationWarning, match=msg):
-        result = table.to_pandas()
+    result = table.to_pandas()
```

Comment:
In several of those cases I removed the warning assert, and added a general filterwarnings mark (at the test or top of the file), because 1) there is no need to "test" that the warning is raised in all those cases, and 2) once the PR to fix this on the pyarrow side is merged, all those cases will start failing anyway